docs/home/howto/OS_PATCHING.md

```diff
@@ -100,3 +115,51 @@ Available options:
 ## Additional information
 
 Red Hat provides notifications about security flaws that affect its products in the form of security advisories. For more information, see [here](https://access.redhat.com/security/updates/advisory).
+
+### Ubuntu
+
+For automated security patches Ubuntu uses unattended-upgrade facility. By default it runs every day. To verify it on your system, execute:  
+`dpkg --list unattended-upgrades`
+`cat /etc/apt/apt.conf.d/20auto-upgrades | grep Unattended-Upgrade`
+
+For information how to change Unattended-Upgrade configuration, see [here](https://github.com/mvo5/unattended-upgrades/blob/master/README.md).
+
+The following steps will allow you to perform an upgrade manually.  
+
+Update your local repository cache:  
+`sudo apt update`
+
+Verify if epirepo is disabled and official repositories are enabled:  
```

Comment:
@rafzei is that possible to attach example output for user to compare?

Reply:
The output is quite big so I wouldn't put this in the documentation. Second thing is that users can disable some repositories and leave e.g. only security-related so It's hard to compare. I can split this to:
 - verify if epirepo is disabled  (apt-cache policy | grep epirepo),
 - verify if all your intended repositories are enabled (apt-cache policy).

Reply:
sounds good! 👍